packages/gguf/src/types.ts

```diff
@@ -29,6 +29,13 @@ export enum GGMLQuantizationType {
 	IQ3_S = 21,
 	IQ2_S = 22,
 	IQ4_XS = 23,
+	I8 = 24,
```

Comment:
@CISC could you paste here in the comments the source from ggml repo for these newly added types?

Reply:
Done